node/grpc/utils.go

```diff
@@ -73,20 +75,34 @@ func GetBlobHeaderFromProto(h *pb.BlobHeader) (*core.BlobHeader, error) {
 		X: *commitX,
 		Y: *commitY,
 	}
+
+	if !(*bn254.G1Affine)(commitment).IsInSubGroup() {
```

Comment:
should these go into the request validation so it can be rejected as soon as possible?

Reply:
i can make the call to "GetBlobHeaderFromProto" in the validation phase

Reply:
fixed, please double check